src/pybind/mgr/dashboard/frontend/src/app/core/auth/login/login.component.html

```diff
@@ -40,6 +40,7 @@
                  [attr.aria-invalid]="password.invalid"
                  aria-labelledby="password"
                  class="form-control pl-4"
+                 autocomplete="new-password"
```

Comment:
Same for this field. This is supposed to be filled with your password, not a new password.